site/docs/README.md

```diff
@@ -13,6 +13,7 @@ hero:
     - can do anything except dishes.
     - easy peasy lemon squeezY.
     - billions and billions served.
+    - powered by obsession.
```

Comment:
"Ditenagai oleh obsesi", "Ditenagai oleh ambisi", "Ditenagai/didorong/disokong/didukung?", "Ambisi/Obsesi???" @fadzikri

Reply:
Better do this comments in `ID` file :D